libs/remix-ui/editor/src/lib/remix-ui-editor.tsx

```diff
@@ -75,6 +75,25 @@ export interface EditorUIProps {
 
 export const EditorUI = (props: EditorUIProps) => {
   const [, setCurrentBreakpoints] = useState({})
+  const defaultEditorValue = `
+  \t\t\t\t\t\t\t ____    _____   __  __   ___  __  __   ___   ____    _____ 
+  \t\t\t\t\t\t\t|  _ \\  | ____| |  \\/  | |_ _| \\ \\/ /  |_ _| |  _ \\  | ____|
+  \t\t\t\t\t\t\t| |_) | |  _|   | |\\/| |  | |   \\  /    | |  | | | | |  _|  
+  \t\t\t\t\t\t\t|  _ <  | |___  | |  | |  | |   /  \\    | |  | |_| | | |___ 
+  \t\t\t\t\t\t\t|_| \\_\\ |_____| |_|  |_| |___| /_/\\_\\  |___| |____/  |_____|\n\n
+  \t\t\t\t\t\t\tKeyboard Shortcuts:\n
+  \t\t\t\t\t\t\t\tCTRL + S: To compile current contract\n
```

Comment:
The line about Compile and the one about Compile & Run are not next to each other.  On one had that seems OK -because nearly everyone can use the compile shortcut.  But on the other hand - they are both about compiling.  Let's leave it as you have it - unless you think otherwise.